rules/common_settings.bzl

```diff
@@ -12,6 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# buildifier: disable=skylark-docstring
+# buildifier: disable=skylark-comment
```

Comment:
These comments are not required anymore, the latest version of buildifer shouldn't warn about the usage of this url anymore.

Reply:
Done.